src/robot-arm-element.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { html } from 'lit';
+import './robot-arm-element';
+
+export default {
+  title: 'Robot Arm',
+  component: 'wokwi-robot-arm',
```

Comment:
How about [wokwi-robotic-arm](https://en.wikipedia.org/wiki/Robotic_arm)? Not being a native English speaker, I'm not sure if there's an actual difference, so I'd love to hear the thoughts of someone whose a native English speaker.

Reply:
Works for me